### PR TITLE
Adding Dataflow xml pubsub to gcs example

### PR DESCRIPTION
**Pull request by atognolag**
# Additions:
1. Dataflow xml pubsub to gcs example and corresponding code and docs
# Changes:
1. Adding configuration command to configure Docker for gcr.io integration to the **CONTRIBUTING.md** file
2. The main **README.md** has been updated to include this example

### CODE REVIEW

File: examples/dataflow-xml-pubsub-to-gcs/python/beamPubSubXml2Gcs.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+            # Beam binds the publish time returned by the Pub/Sub server for
+            # each message to the element's timestamp parameter, accessible via
+            # `DoFn.TimestampParam`.
+            # https://beam.apache.org/releases/pydoc/current/apache_beam.io.gcp.pubsub.html#apache_beam.io.gcp.pubsub.ReadFromPubSub
+            # https://cloud.google.com/pubsub/docs/stream-messages-dataflow#set_up_your_pubsub_project
+            | "Read from Pub/Sub" >> io.ReadFromPubSub(topic=input_topic)
+            | "Parse XML tags and attributes" >> ParDo(ParseXML())
+            | "Window into" >> GroupMessagesByFixedWindows(window_size,
+                                                           num_shards)
+            | "Write to GCS" >> ParDo(WriteToGCS(output_path))
```
Comment (aymanfarhat) [inline examples/dataflow-xml-pubsub-to-gcs/python/beamPubSubXml2Gcs.py:151]: I would replace this part and use the built in [WriteToText](https://beam.apache.org/releases/pydoc/2.41.0/apache_beam.io.textio.html#apache_beam.io.textio.WriteToText) transform. With this transform you should be able to scale writes better than using file systems per pCollection item. 

You might need to add a step before for pre-processing / grouping of data depending on the structure of the data you have in the PCollection

File: examples/dataflow-xml-pubsub-to-gcs/python/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (aymanfarhat) [inline examples/dataflow-xml-pubsub-to-gcs/python/.vscode/launch.json:1]: Add license here as well please.

**Review by aymanfarhat — CHANGES_REQUESTED** [review]
Thank you @atognolag, I added a couple of comments that need to be tackled before final approval.

**Comment by atognolag** [comment]
Hi @aymanfarhat please review the new changes and resolve the comments if the changes look good to you!

File: examples/dataflow-xml-pubsub-to-gcs/python/.vscode/launch.json (hunk 2 above)
Comment (atognolag) [reply examples/dataflow-xml-pubsub-to-gcs/python/.vscode/launch.json:1]: Addressed in latest commit, please review

File: examples/dataflow-xml-pubsub-to-gcs/python/beamPubSubXml2Gcs.py (hunk 1 above)
Comment (atognolag) [reply examples/dataflow-xml-pubsub-to-gcs/python/beamPubSubXml2Gcs.py]: Addressed in latest commit, please review

**Review by aymanfarhat — APPROVED** [review]
LGTM - Thank you for your contribution!

**Pull request merged**